_includes/module-loading/bundling-webpack-cjs-npm_editor.md

```diff
@@ -0,0 +1,49 @@
+1. Create a new source file for importing the required components from {{site.productname}} and configuring the editor.
+
+    {% include module-loading/bundling-required-components.md %}
+
+    Example `src/editor.js`
+
+    ```js
+    /* Import TinyMCE */
+    var tinymce = require('tinymce/tinymce');
+
+    /* Default icons are required for TinyMCE 5.3 or above. Also import custom icons if applicable */
+    require('tinymce/icons/default');
+
+    /* A editor theme (required) - customize the editor appearance by creating a 'skin' */
+    require('tinymce/themes/silver');
+
+    /* Import the editor skin - replace with a custom skin if applicable. */
+    require('tinymce/skins/ui/oxide/skin.css');
+
+    /* Import plugins - include the relevant plugin in the 'plugins' option. */
+    require('tinymce/plugins/advlist');
+    require('tinymce/plugins/code');
+    require('tinymce/plugins/emoticons');
+    require('tinymce/plugins/emoticons/js/emojis');
+    require('tinymce/plugins/link');
+    require('tinymce/plugins/lists');
+    require('tinymce/plugins/table');
+
+    /* Import content CSS */
+    var contentUiCss = require('tinymce/skins/ui/oxide/content.css').toString();
```

Comment:
Why do we do `toString` here but in the ES6 version we do it below when concatenating the values? They should probably be consistent.

Reply:
Fixed in [`d7580f1` (#2035)](https://github.com/tinymce/tinymce-docs/pull/2035/commits/d7580f10a1419e1c9e16fbd5baeac863501719ab)